smartsim/_core/_install/buildenv.py

```diff
@@ -46,8 +49,6 @@
 #       to remove
 # https://setuptools.pypa.io/en/latest/pkg_resources.html
 
-import pkg_resources
-from pkg_resources import packaging  # type: ignore
```

Comment:
Can we `# fmt: off`/`# fmt: on` (and/or the `isort` equivalent) around these imports? I just want to make sure they stay below the banner comment addressing the a potential issue with relying on `pkg_resources`